openmc/geometry.py

```diff
@@ -39,11 +39,11 @@ class Geometry:
 
     """
 
-    def __init__(self, root=None):
+    def __init__(self, root : openmc.UniverseBase = None, merge_surfaces: bool = False, surface_precision: int = 10):
```

Comment:
Do we have a policy on how to write method/constructor signatures with type annotations yet? With type annotations (and default values in particular) I find this style a bit hard to read unless there are only 1 or 2 parameters. Since this line seems to be getting a bit longer than the 90ish rule, perhaps 1 argument per line, but I'd be curious to hear other perspectives. I think my preference is one per line with whitespace up the the ( but there are good arguments for other styles out there

Reply:
I've changed the code to multi line args and it does look better imo but I'm always keen automate this stuff and adopt whatever flake8 or black decides